client/my-sites/sharing/buttons/preview-action.jsx

```diff
@@ -1,52 +1,44 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	classNames = require( 'classnames' );
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+import { omit, startsWith, endsWith } from 'lodash';
 
-module.exports = React.createClass( {
-	displayName: 'SharingButtonsPreviewAction',
+const SharingButtonsPreviewAction = ( props ) => {
```

Comment:
That would make sense if we remove the `omit`, but we're keeping it - see https://github.com/Automattic/wp-calypso/pull/8779/files#r84030371